server/lib/report-manager.js

```diff
@@ -101,7 +99,7 @@ class ReportManager
         subject = $t('server.email.report.subject', { frequency });
         subject = `Kresus - ${subject}`;
 
-        let today = moment().format('DD/MM/YYYY');
+        let today = moment().format('L');
```

Comment:
The moment lib is not loaded in this module. It is loaded throught the helpers. Maybe this should also be done for all other module using moment. And yes, it does work